docs/advanced/persistence-and-collaboration.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Persistence and Collaboration
+---
+
+## Introduction
+
+By default `Variable` state is purely client-side - by default its source of truth is the browser memory.
```

Comment:
By default `Variable` state is purely client-side -> By default a `Variable`'s state is purely client-side 

Also this sentence feels a bit awkward maybe just putting a connector word:

.. is purely client-side, meaning its source of truth is the browser memory.